lib/jsonapi/serializer.ex

```diff
@@ -79,12 +79,12 @@ defmodule JSONAPI.Serializer do
   @spec encode_relationships(Conn.t(), document(), tuple(), list()) :: tuple()
   def encode_relationships(conn, doc, {view, data, _, _} = view_info, options) do
     view.relationships()
-    |> Enum.filter(&assoc_loaded?(Map.get(data, elem(&1, 0))))
```

Comment:
Annotation: If we want to put this feature behind a config flag, we should place the config flag here, to continue filtering-out unloaded relationships.